### ISSUE TITLE
Duplicating exhibits results in multiple copies being spawned

### ISSUE THREAD

**Issue by rondauva** [performant-software/Neatline#473]
see also: https://forum.omeka.org/t/neatline-duplicate-function-problems/9115

User reports that clicking the Duplicate button on an exhibit on the Neatline Exhibits Browse screen results in ~100 copies being spawned. All copies initially report that they are empty on the Browse Exhibits page. Omeka Classic 2.7, Neatline version TBD.

Testing here on workshop.neatline.org with Omeka 2.6.1 and Neatline 2.6.1, I get only 1 duplicate, as expected, but it shows that it has 0 items on the Browse Exhibits screen. When opening the Neatline editor, all items are duplicated, but aren't functional. Attempting to edit, changes the values, but doesn't immediately result in changed appearance or function. Once items are saved, and the exhibit editor is closed, the Browse exhibits page reports the correct # of items in the exhibit. Opening the Neatline editor and checking each item, the changes made were saved from previous editing session, and exhibit functionality seems restored.

Have not tested with Omeka 2.7 or the latest release of Neatline. Will update if anything is different with that configuration.

**Comment by Dr-Hutchinson** [comment]
I'm the user facing the difficulty, many thanks for the assistance. Running Neatline 2.6.1.

**Comment by jeremyboggs** [comment]
@Dr-Hutchinson I've not looked into this yet. And I'm skeptical it will help, but do try to update to the latest patch I just published, [Neatline 2.6.3](https://github.com/scholarslab/Neatline/releases/tag/v2.6.3)

**Comment by jamiefolsom** [comment]
Hi @Dr-Hutchinson, and @rondauva and @jeremyboggs. Thanks for this report. So it sounds like there are two separate issues, the one with Omeka 2.7 resulting in too many duplicates, and the one with Omeka 2.6.x with non-functional exhibits or records, is that accurate? 

If you can confirm those details, and provide any others you can (perhaps data or screenshots from the Omeka install, exhibits, and/or records in question, or steps to reproduce), that would be helpful.

**Comment by jeremyboggs** [comment]
@jamiefolsom I suspect the release I just tagged will help with the non-functional exhibits/records. But yes, it seems like two separate issues. There are no duplication issues that we can see with Omeka 2.6.x, so it might just be an issue with 2.7.

**Comment by AntjeAnderson** [comment]
I have just begun to work with Omeka and Neatline, so I have Omeka 2.7 and Neatline 2.6.3., and my attempt to duplicate my one and only Neatline Exhibit resulted in 18 pages of empty exhibits! So the problem has not gone away. My question is very basic: how do I delete these? I am working on  Reclaim Hosting as my web server, and I cannot figure out where these empty files would be located so I can delete them. In fact, I cannot figure out where my Neatline files go at all. Please help me figure out what to do about these hundreds of duplicates. Thank you.

**Comment by smithkevin1** [comment]
@AntjeAnderson I just experienced the same thing and have the same problem (Neatline 2.6.2 and Omeka 2.7). Is there a way to bulk delete exhibits on the back end?

**Comment by AntjeAnderson** [comment]
@smithkevin1 , I am not 100% sure because I had help from a student. I will try to get her on here. I think you still just need to go into individual lines in the database sub-section called for Neatline Exhibits through PhPMyAdmin and check the empty ones off to be deleted. THAT SAID, I did manage to find a fix and I can now duplicate without the spawning. I was going to let @jeremyboggs post after he maybe double-checks to confirm, but if you follow these reclaim hosting community instructions for fixing the config in Reclaim Hosting, the duplication works. https://community.reclaimhosting.com/t/setting-the-php-cli-path-in-omeka-classic/231.

**Comment by smithkevin1** [comment]
@AntjeAnderson Yes! Going through PhPMyAdmin worked. Thanks so much. It had spawned over 850 duplicates and I had no idea how to delete them.

**Comment by AntjeAnderson** [comment]
@smithkevin1, so glad. Again, by following the Reclaim hosting user's instructions, I've been able to fix this problem and I now have a functioning duplicate of my exhibit (if you are still trying for that). Since copying individual Neatline Records is still giving me grief because of the way the geographic information is stored, I am relieved this works.
